pbs/pbsresponse.go

```diff
@@ -16,6 +16,8 @@ type PBSBid struct {
 	// it helps publishers and bidders identify and communicate about malicious or inappropriate ads.
 	// This project simply passes it along with the bid.
 	Creative_id string `json:"creative_id,omitempty"`
+	// CreativeMediaType shows whether the creative is a video or banner.
+	CreativeMediaType MediaType `json:"media_type,omitempty"`
```

Comment:
I think this type should be a `string` instead `MediaType`.  Also I don't see the value being set in any of the bid responses in the adapters (around line 192 of appnexus.go for example).